addon/array/filter-by.js

```diff
@@ -7,11 +7,15 @@ const defaultValue = [];
 export default createClassComputed(
   [false, true, false],
   (array, key, value) => {
-    return computed(normalizeArrayKey(array, [key]), value, (array, value) => {
+    let args = [normalizeArrayKey(array, [key])];
+    if (value) {
+      args.push(value);
+    }
```

Comment:
The problem with checking `value` for truthiness (or presence even), is that you're not doing 100 % the same as checking the actual length of `arguments`.

That's why I wasn't able to finish the PR in the first place: :grinning: 

But if that is not an issue for you, we can let that nitpick slide and I am :+1: on this.